fair/forms.py

```diff
@@ -1,9 +1,15 @@
 from django import forms
 
-from .models import FairDay, LunchTicket
+from .models import FairDay, LunchTicket, Fair, LunchTicketTime
 
 
 class LunchTicketForm(forms.ModelForm):
+	currentFair = Fair.objects.filter(current=True).first()
+	day = forms.ModelChoiceField(queryset = FairDay.objects.filter(fair = currentFair), required = True)
+	allDays = []
+	for fairDay in FairDay.objects.filter(fair = currentFair):
+		allDays.append(fairDay)
+	time = forms.ModelChoiceField(queryset = LunchTicketTime.objects.filter(day__in = allDays), required = True)
```

Comment:
The time should preferrably not be required since we probably set all the lunch times later on and that it is sometimes needed to leave it empty since the person should not have a fixed lunch time. In the model of the LunchTicket time can be null or blank so should just be to remove this required=True here.